package.json

```diff
@@ -53,9 +54,15 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint && yarn test"
+      "pre-commit": "lint-staged",
+      "post-commit": "git update-index --again",
+      "pre-push": "yarn test"
     }
   },
+  "lint-staged": {
+    "*.js": ["prettier --write", "eslint --fix --max-warnings 0", "git add"],
```

Comment:
@olslash thoughts on `eslint --fix`?
- Pro: I don't have to chase down unambiguous-to-fix errors
- Con: Additional code change (in addition to prettier) that the committer might not be expecting

Reply:
i don't trust eslint --fix.

I'm ok with stylelint fixing because that's going to be mostly formatting, but eslint messing with code is not something I want to happen, especially when i'm in the middle of a commit and verifying what it's changed is out of my control.

Reply:
Works for me